src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/command/flow/ingress/of/IngressInstallFlowModFactory.java

```diff
@@ -61,6 +63,21 @@ public IngressInstallFlowModFactory(
         return instructions;
     }
 
+    @Override
+    protected List<OFInstruction> makeIngressFlowLoopInstructions(FlowEndpoint endpoint) {
+        List<OFAction> actions = new ArrayList<>();
+        if (endpoint.getInnerVlanId() != 0) {
+            OfAdapter.INSTANCE.makeVlanReplaceActions(of, Collections.singletonList(endpoint.getOuterVlanId()),
```

Comment:
The return value of `OfAdapter.INSTANCE.makeVlanReplaceActions()` is not used anywhere.

Reply:
Fixed